test/parallel/test-console.js

```diff
@@ -138,6 +138,20 @@ console.timeEnd();
 console.time(NaN);
 console.timeEnd(NaN);
 
+// make sure calling time twice without timeEnd doesn't reset the timer.
+console.time('test');
+const time = console._times.get('test');
+setTimeout(() => {
+  console.time('test');
+  assert.deepStrictEqual(console._times.get('test'), time);
+  common.expectWarning(
```

Comment:
I'm a bit surprised this actually works, shouldn't it be set before calling `console.time('test')`?

Reply:
it's set before the second call, which is when it should emit